src/core/frontend/emu_window.h

```
@@ -156,6 +157,14 @@ class EmuWindow {
         return window_info;
     }
 
+    /**
+     * Returns system information about the drawing area.
+     * Unsafe version of above for MoltenVK to get pointer
+     */
+    WindowSystemInfo& GetWindowInfoRef() {
```

This should also have a const getter as well.

---

Apple compiles had one of the unused parameter warnings throw an error instead of a warning past a certain version, but this seems to have been fixed. Reverted.

---

Changed to getting raw surface pointer